### ISSUE TITLE
additional reward rains

### ISSUE THREAD

**Issue by ghost** [phetsims/equality-explorer#129]
For phetsims/QA/issues/131. This is kind of nitpicky, but I'm confused as to why apples and oranges are appearing in the additional reward for level 1:
![lvl1](https://user-images.githubusercontent.com/31285590/41870800-4c65d23a-787a-11e8-819c-cae905006108.png)

Here's level 2:
![lvl2](https://user-images.githubusercontent.com/31285590/41870814-5919b8c0-787a-11e8-87f7-4227fbf98668.png)

Here's level 3:
![lvl3](https://user-images.githubusercontent.com/31285590/41870820-6177199a-787a-11e8-97e7-f77361f3caf2.png)

I'm also confused as to why there are dogs and cats in level 4:
![lvl4](https://user-images.githubusercontent.com/31285590/41870862-7d6c2424-787a-11e8-88c1-f965f7c3b7f0.png)

Was the idea just to get some variety in the additional rewards?

**Comment by amanda-phet** [comment]
>Was the idea just to get some variety in the additional rewards? 

Yes, that is correct. We went through some different iterations, but really wanted it to rain cats and dogs on the last level :) Happy to take suggestion to improve or making the reward nodes less confusing!

**Comment by ghost** [comment]
This might be a lot of work, but it would be cool to see PhET girl be "upgraded" in the "Keep Going / New Level" dialogue. Maybe she gets a star for each level she completes? Or something like that. It would be nice to see some progression beyond the "raining" items changing.

**Comment by amanda-phet** [comment]
I think for this purpose, it is fine to leave phet girl as she is since the levels aren't exactly cumulative (and the same dialog is used in make a ten, and that sim has 10 levels!). 

But if you think the rewardnode objects are confusing, do you have ideas for how to improve those?

**Comment by ghost** [comment]
I think something like this would make more sense:

Lvl1: integer variable terms + integer constant terms
Lvl2: integer variable terms + integer constant terms + stars
Lvl3: integer and rational variable terms + integer and rational constant terms + smileys
Lvl4: integer variable terms + integer constant terms + stars + smileys

Might be more boring, though. Again, sorry to nitpick. I just thought it was weird seeing the oranges, apples, dogs, and cats in the Solve It! screen.

**Comment by phet-steele** [comment]
I talked to @lmulhall-phet on Slack and we came to the conclusion to leave the sim as is. @lmulhall-phet thought it was weird to see objects from other screens in the game screen, but this is not the first sim to do this. Similarly, as @lmulhall-phet states:

> Might be more boring, though.

This is exactly what we wanted to avoid in #63. The current way spices up the game and adds more clear distinction between already very similar levels.
